modules/fluid_structure_interaction/src/auxkernels/WaveHeightAuxKernel.C

```diff
@@ -0,0 +1,38 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "WaveHeightAuxKernel.h"
+
+registerMooseObject("FluidStructureInteractionApp", WaveHeightAuxKernel);
+
+InputParameters
+WaveHeightAuxKernel::validParams()
+{
+  InputParameters params = AuxKernel::validParams();
+  params.addClassDescription("Calculates the wave heights given pressures.");
+  params.addRequiredCoupledVar("pressure", "pressure variable");
+  params.addRequiredParam<Real>("dens", "fluid density");
+  params.addRequiredParam<Real>("grav", "acceleration due to gravity");
```

Comment:
there really is no good reason to aggressively abbreviate parameter names. Why not just use `density` and `gravity` here?